DependencyInjection/HttplugExtension.php

```diff
@@ -106,6 +106,16 @@ private function configureClients(ContainerBuilder $container, array $config)
                 $container->setAlias('httplug.client.default', 'httplug.client.'.$first);
             }
         }
+
+        $mockClass = 'Http\Mock\Client';
```

Comment:
lets use the ::class pseudoconstant here. and make the use statement say `use Http\Mock\Client as MockClient;` to make things more readable.